geopandas/io/tests/test_arrow.py

```diff
@@ -681,35 +681,41 @@ def test_write_read_default_crs(tmpdir, format):
 @pytest.mark.parametrize(
     "format,version", product(["feather", "parquet"], [None] + SUPPORTED_VERSIONS)
 )
-def test_write_spec_version(tmpdir, format, version):
+def test_write_deprecated_version_parameter(tmpdir, format, version):
```

Comment:
Shouldn't we keep the original version of this test as well?

Reply:
Yes!  I must have modified in place rather than in a copy as I had intended; good catch!